sklearn/decomposition/_dict_learning.py

```diff
@@ -239,31 +241,31 @@ def sparse_encode(X, dictionary, *, gram=None, cov=None,
         the reconstruction error targeted. In this case, it overrides
         `n_nonzero_coefs`.
 
-    copy_cov : boolean, optional
+    copy_cov : boolean, default=True
         Whether to copy the precomputed covariance matrix; if False, it may be
         overwritten.
 
-    init : array of shape (n_samples, n_components)
+    init : array of shape (n_samples, n_components), default=None
         Initialization value of the sparse codes. Only used if
         `algorithm='lasso_cd'`.
 
-    max_iter : int, 1000 by default
+    max_iter : int, default=1000
         Maximum number of iterations to perform if `algorithm='lasso_cd'` or
         `lasso_lars`.
 
-    n_jobs : int or None, optional (default=None)
+    n_jobs : int or None, default=None
```

Comment:
if the default is None, it doesn't have to be listed, i.e. you could write `: int, default=None`